src/js/import.js

```diff
@@ -195,8 +220,17 @@
         $(".import-extraHeaders").each(function() {
           selected.push($(this).val());
         });
-        $(".import-leave").each(function() {
-          selected.push($(this).val());
+        $(".import-intervention").each(function() {
+         selected.push($(this).val());
+        });
+        $(".import-effect").each(function() {
+         selected.push($(this).val());
+        });
```

Comment:
Not 100% sure of the structure here, but perhaps you can do something like: 
```js
var selected = $('.import-serp-select-wrapper').find('select').map((i, e) => e.value).toArray()
```
Find is pretty cool, docs here: https://api.jquery.com/find/